steam/ext/csgo/state.py

```diff
@@ -216,10 +216,9 @@ def handle_players_profile(self, msg: GCMsgProto[cstrike.PlayersProfile]):
     @register(Language.Client2GCEconPreviewDataBlockResponse)
     def handle_client_preview_data_block_response(self, msg: GCMsgProto[cstrike.Client2GcEconPreviewDataBlockResponse]):
         # decode the wear
-        with utils.StructIO() as io:
-            io.write_u32(msg.body.iteminfo.paintwear)
-            item = utils.get(self.backpack, id=msg.body.iteminfo.itemid)
-            item.paint_wear = io.read_f32()
+        item = msg.body.iteminfo
```

Comment:
This isn't used
```suggestion
```

Reply:
I can't commit that - item variable is used in next few lines of code. Do you mean i need change item variable to msg.body.iteminfo?
```python
  packed_wear = struct.pack(">l", msg.body.iteminfo.paintwear)
  msg.body.iteminfo.paintwear = struct.unpack(">f", packed_wear)[0]
  self.dispatch("inspect_item_info", msg.body.iteminfo)
```

Reply:
I'm actually going to add an inspected item class or something so people can get more attributes like position and have a similar structure to the other classes I have in the works